appengine/flexible/storage/noxfile_config.py

```diff
@@ -22,7 +22,7 @@
 
 TEST_CONFIG_OVERRIDE = {
     # You can opt out from the test for specific Python versions.
-    "ignored_versions": ["2.7"],
+    "ignored_versions": ["2.7", "3.6", "3.7", "3.8", "3.9", "3.10"],
```

Comment:
Just confirming- you **only** want to test this in 3.11, and no earlier versions?

Reply:
Yes, this is effectively a python 3.11 app in prod.